src/SingleProject/Resizetizer/src/ResizetizeImages.cs

```diff
@@ -101,7 +101,7 @@ public override System.Threading.Tasks.Task ExecuteAsync()
 
 				// Add DPI info to the itemspec so we can use it in the targets
 				attr.Add("_ResizetizerDpiPath", img.Dpi.Path);
-				attr.Add("_ResizetizerDpiScale", img.Dpi.Scale.ToString());
+				attr.Add("_ResizetizerDpiScale", img.Dpi.Scale.ToString("0.0"));
```

Comment:
This should pass in `CultureInfo.InvariantCulture` to avoid culture-specific formatting. If you run this code on a non-English-US system, this could produce unexpected results because the `0.0` format might produce something like `123,4` on a French system.

That is, unless you happen to know that MSBuild always runs in en-US regardless of the OS settings (I can't find any docs to this effect, only [this unanswered StackOverflow question](https://stackoverflow.com/questions/57803349/what-is-the-default-culture-of-an-msbuild-task)).

Reply:
Good call!